### ISSUE TITLE
Players Cannot Open Things

### ISSUE THREAD

**Issue by gabefoster** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#101]
When a player tries to open a door, GMs or the owner (with the GM role selected) should be notified, so as to grant the player access. At present, this works for those assigned the GM role, but not for owners (with the GM role selected). Verified for chests and doors.

**To Reproduce**
1. Create a campaign with a board with a door.
2. Have a player load into that board.
3. Create a character for that player, and assign them permissions for that character.
4. Have that player try to open the door you created. 
**Screenshots/Videos/Twitch Clips**
Sorry, I don't have video recording set up right now. Feel free to ping me with questions!

**Additional context**
Basically I think board campaign owners aren't being recognized as having the "GM" role for the purposes of requests to open things. Probably just a missing flag somewhere.

**Comment by gabefoster** [comment]
Update: It's now working, but says a different player is asking to open the chest/door. Maybe this is an indexing issue / off by 1 error? Looking for the nth character when it should be the (n-1)th?
